packages/react-components-pro/package.json

```
@@ -30,6 +30,7 @@
     "@vaadin/charts": "24.5.0-beta1",
     "@vaadin/cookie-consent": "24.5.0-beta1",
     "@vaadin/crud": "24.5.0-beta1",
+    "@vaadin/dashboard": "24.6.0-alpha0",
```

Let's rebase and update to alpha1.
```suggestion
    "@vaadin/dashboard": "24.6.0-alpha1",
```

---

Ah, still need to update lock file

---

Done